### PR TITLE
Add Neighbors

### PR DESCRIPTION
**Pull request by zalo**
@LukPopp0 This PR exposes Cell Neighbors, which happen to be the edges of the Delaunay Tetrahedralization (the dual of the Voronoi Graph).

This can be used for tetrahedralizing point clouds, and eventually, meshes.

The typescript definitions seem to have changed a bit due to the upgraded version of Emscripten this PR's sister PR is using: https://github.com/LukPopp0/Voro-Emscripten/pull/1

### CODE REVIEW

**Review by LukPopp0 — APPROVED** [review]
Thanks for updating this as well! This should be good 👍🏻 I needed to update the emscripten version and changed flags anyway.
I'll let you merge it whenever you want.

**Comment by zalo** [comment]
Thanks @LukPopp0 !    

I think you have to be the one to hit the merge button since it’s your repo 😅

One of my secondary goals for this is to let there be more tetrahedralization javascript frameworks, to help with creating models for FEM and physics simulations like this: https://zalo.github.io/TetSim/

Also, I might come back and expose functions for plane cutting and setting the radius of each voronoi cell. 👀

**Pull request merged**
